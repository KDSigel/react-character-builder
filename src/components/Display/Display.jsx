import React from 'react'

export default function Display({allCatchphrases}) {
    return (
        <div>
            Catchphases: {allCatchphrases.map((oneCatchphrase) => (
                <div key={oneCatchphrase}>{oneCatchphrase}</div>
            ))}
        </div>
    )
}
