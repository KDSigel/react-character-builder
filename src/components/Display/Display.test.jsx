import {render} from '@testing-library/react';
import Display from './Display';

it('renders display and looks at html stuff', () => {

    const {container} = render(<Display allCatchphrases={['sup?', 'nah.', 'Why Now?']} />)
    expect(container).toMatchSnapshot();
}
)