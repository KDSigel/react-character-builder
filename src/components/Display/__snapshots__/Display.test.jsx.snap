// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders display and looks at html stuff 1`] = `
<div>
  <div>
    Catchphases: 
    <div>
      sup?
    </div>
    <div>
      nah.
    </div>
    <div>
      Why Now?
    </div>
  </div>
</div>
`;
